app/views/ctc/questions/confirm_payment/edit.html.erb

```diff
@@ -16,6 +16,16 @@
         <p class="spacing-below-15"><%= number_to_currency(@third_stimulus_amount) %></p>
       </div>
     <% end %>
+    <% if @eitc_amount %>
```

Comment:
Non-blocking: What's your philosophy on checking within the ERB if `Flipper.enabled?(:eitc)`? What you've written is fine since this value is nil if EITC is disabled. I prefer adding `Flipper` checks everywhere we render EITC stuff within the ERB, and very-few-but-just-enough checks in the backend. Feel free not to answer if you're busy.